packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -256,13 +306,31 @@ const Route = ({
   ) : null;
 
   const attributesQueryBuilder =
-    route.action?.type === "eventTypeRedirectUrl" ? (
+    route.action?.type === "eventTypeRedirectUrl" && isTeamForm ? (
       <div className="mt-4">
-        {isTeamForm && (
```

Comment:
Instead of having `isTeamForm` conditional just for the text, now we have it for entire attributesQueryBuilder.
This doesn't change the behaviour because `attributesQueryBuilderConfig` is null when `isTeamForm` is false and thus nothing is rendered still.

This change only makes the code more readable.